recipes/poshlib/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "1.3.002":
```

Comment:
This is not exactly the version from the origin source, it is `1.3.002` plus some additions, we should use `cci.<date>` instead.

Reply:
The source codes have not been changed. Only the readme and the license have been modified.

Reply:
@jgsogo The author uses "internal version" if you read the readme, but I agree with you, it's fragile in terms of tracking. I just opened an issue asking for a formal tag, so we can download the specific version: https://github.com/PhilipLudington/poshlib/issues/1

Reply:
Now we have a real tag: https://github.com/PhilipLudington/poshlib/releases/tag/v1.3.002

Reply:
Thanks !